From: Matthew Flatt <mflatt@racket-lang.org>
Date: Mon, 11 Mar 2024 08:05:33 -0600
Subject: repair for embed-boot commit

A mangled commit meant that the code did not match the commit message.
---
 src/cs/c/embed-boot.rkt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/cs/c/embed-boot.rkt b/src/cs/c/embed-boot.rkt
index 7b8860e..24ed404 100644
--- a/src/cs/c/embed-boot.rkt
+++ b/src/cs/c/embed-boot.rkt
@@ -127,8 +127,7 @@
      (define terminator-len (bytes-length terminator))
 
      (define big-endian?
-       (if (and target
-                (not (equal? target (symbol->string (system-type 'target-machine)))))
+       (if target
            (case target
              [("tppc32osx" "ppc32osx"
                           "tppc32le" "ppc32le"
